packages/core/__tests__/global.js

```diff
@@ -2,15 +2,24 @@
 import 'test-utils/dev-mode'
 import * as React from 'react'
 import { render, unmountComponentAtNode } from 'react-dom'
-import { Global, keyframes, css } from '@emotion/core'
+import { Global, keyframes, css, withEmotionCache } from '@emotion/core'
 
-beforeEach(() => {
-  // $FlowFixMe
-  document.head.innerHTML = ''
+const ClearCache = withEmotionCache((props, cache) => {
+  cache.sheet.flush()
+  return null
+})
```

Comment:
I'd rather create a new cache in each test and pass it down with the CacheProvider and clear everything in the head in beforeEach